mllib/src/main/scala/org/apache/spark/ml/feature/StandardScaler.scala

```diff
@@ -29,7 +29,18 @@ import org.apache.spark.sql.types.{StructField, StructType}
 /**
  * Params for [[StandardScaler]] and [[StandardScalerModel]].
  */
-private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol
+private[feature] trait StandardScalerParams extends Params with HasInputCol with HasOutputCol {
+  val withMean: BooleanParam = new BooleanParam(this, 
```

Comment:
Add doc please, including group param.  Feel free to copy doc from spark.mllib API